subsys/bootloader/bl_validation/bl_validation.c

```diff
@@ -338,6 +355,8 @@ static bool validate_firmware(uint32_t fw_dst_address, uint32_t fw_src_address,
 		return false;
 	}
 
+	PRINT("Trying to get Firmware version");
```

Comment:
All the other PRINT calls have "\n\r" at the end, do we want it here also?

Reply:
Damn I knew I left this one in it haunted me :D I'm removing it.